controller/src/main/java/io/pravega/controller/server/rest/resources/StreamMetadataResourceImpl.java

```diff
@@ -86,6 +98,15 @@ public void createScope(final CreateScopeRequest createScopeRequest, final Secur
             return;
         }
 
+        try {
```

Comment:
This is a general comment on most methods of this file (though outside the changes of this PR).  The security context is injected in each method but never used as authentication is done using the explicit `authenticate` method, and not using anything from the security context.

If the security context is not used, consider removing for clarity?

Another thing was: maybe in the future (outside this PR), using a filter based approach could remove some code duplication (calls to `authenticate`) inside the methods of this file.     Moving the call to authenticate inside a filter that is applied to all these methods in the rest resource could make these a lot lighter by separating business logic from security.

Each method could be annotated with custom annotations depicting which level is required to access the method, and the filter could have access to the request path params and query params, in order to figure out the authz logic (scope name etc.)